packages/native/test/lib/ElementAssertion.test.tsx

```diff
@@ -1,13 +1,40 @@
 import { AssertionError, expect } from "@assertive-ts/core";
-import { render } from "@testing-library/react-native";
+import { fireEvent, render } from "@testing-library/react-native";
+import { useState, useCallback } from "react";
 import {
   View,
   TextInput,
   Text,
+  Modal,
+  Button,
 } from "react-native";
 
 import { ElementAssertion } from "../../src/lib/ElementAssertion";
 
+const SimpleToggleText: React.FC = () => {
```

Comment:
We should avoid using the `React.FC` type, as its function return type is too broad. A named function is usually a better choice, as it also gets hoisted 😬 
```suggestion
function SimpleToggleText(): ReactElement {
```